package.json

```diff
@@ -12,9 +12,10 @@
     "@types/jest": "^26.0.20",
     "@types/node": "^12.20.4",
     "@types/node-sass": "^4.11.1",
-    "@types/react": "^17.0.2",
-    "@types/react-dom": "^17.0.1",
+    "@types/react-router-dom": "^5.1.7",
+    "install": "^0.13.0",
     "node-sass": "^5.0.0",
+    "npm": "^7.6.1",
```

Comment:
Hi @JakubKepak, I think you may have added the "npm" and "install" dependencies by accident

Reply:
I did :(